homeassistant/components/playstation_network/strings.json

```
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "npsso": "NPSSO Token"
+        },
+        "data_description": {
+          "npsso": "Please supply the NPSSO token generated during successful login of your PlayStation Network Account"
+        },
+        "description": "To obtain the NPSSO Token, first login to your playstation account at playstation.com. Then [click here]({npsso_link}) to retrieve your npsso token."
```

```suggestion
        "description": "To obtain your NPSSO token, log in to your [PlayStation account]({psn_link}) first. Then, [click here]({npsso_link}) to retrieve the token."
```
Also use a link to playstation.com